src/seedu/addressbook/commands/HelloCommand.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.commands;
+
+/**
+ * Prints "Hello!".
+ */
+public class HelloCommand extends Command {
+
+    public static final String COMMAND_WORD = "hello";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": says 'Hello!'\n"
```

Comment:
Can consider using `MESSAGE_HELLO` in place of hard coding `'Hello!'` to remove duplication.